pkg/appregistry/adapter.go

```diff
@@ -29,9 +31,13 @@ type apprApiAdapterImpl struct {
 	client *appr.Appregistry
 }
 
-func (a *apprApiAdapterImpl) ListPackages() (appr_models.Packages, error) {
+func (a *apprApiAdapterImpl) ListPackages(namespace string) (appr_models.Packages, error) {
```

Comment:
The more I am thinking about it, I feel we should have the args to functions as fields in apprApiAdapterImpl struct